### PR TITLE
Show a generic message to the developer for admin review activity.

### LINKED ISSUE #3642: Show super-review request on developer page
#3641 hides the super-review request on the developer page entirely. However, we should show the event itself (without the message body) on the developer page as we are sending out an email when an add-on gets admin flagged and so that should be documented on the review activity page as well.

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #3642

### CODE REVIEW

File: src/olympia/amo/log.py [hunk 1]
```diff
@@ -234,7 +234,9 @@ class REQUEST_SUPER_REVIEW(_LOG):
     short = _(u'Super review requested')
     keep = True
     review_queue = True
-    hide_developer = True
+    sanitize = _(u'The addon has been flagged for Admin Review.  Its still in '
+                 u'our review queue, but it will need to be checked by one of '
+                 u'our admin reviewers.')
```
Comment (wagnerand) [inline src/olympia/amo/log.py]: Do we want to add that this means the review could take a bit longer?

Comment (eviljeff) [inline src/olympia/amo/log.py]: I cribbed it from https://github.com/mozilla/addons-server/blob/master/src/olympia/editors/templates/editors/emails/author_super_review.ltxt but tried to make it a bit shorter.  Alternative copy suggestion?

Comment (wagnerand) [inline src/olympia/amo/log.py]: Add: "The review might take longer than usual."

Comment (jvillalobos) [inline src/olympia/amo/log.py]: Also, "it's"

**Pull request merged**